mne/io/meas_info.py

```diff
@@ -60,8 +124,11 @@ class Info(dict):
     bads : list of str
         List of bad (noisy/broken) channels, by name. These channels will by
         default be ignored by many processing steps.
-    ch_names : list of str
+    ch_names : instance of _ChannelNameList (read-only)
```

Comment:
don't expose private object in docstring

I would write

ch_names : non-mutable list-like of str